packages/core/src/entity/Collection.ts

```diff
@@ -47,6 +47,23 @@ export class Collection<T, O = unknown> extends ArrayCollection<T, O> {
     return super.getItems();
   }
 
+  /**
+   * Counts the number of entries in the database
```

Comment:
this sounds a bit misleading, it's count of items of this collection, loaded from database. 

```suggestion
   * Gets the count of collection items from database instead of counting loaded items.
   * The value is cached, use `refresh = true` to force reload it.
```